app/controllers/provider_interface/rejections_controller.rb

```diff
@@ -99,5 +102,9 @@ def check_application_is_rejectable
     def check_feature_flag
       render_404 unless FeatureFlag.active?(:structured_reasons_for_rejection_redesign)
     end
+
+    def wizard_entrypoint_paths
+      [new_provider_interface_application_choice_decision_path]
```

Comment:
We will be changing the spec which tests for R4R wizard cache clearing as part of rolling out the feature so I've omitted a test for this in this PR.